api/v1alpha1/nstemplatetier_types.go

```diff
@@ -22,6 +22,18 @@ type NSTemplateTierSpec struct {
 	// +optional
 	// +mapType=atomic
 	SpaceRoles map[string]NSTemplateTierSpaceRole `json:"spaceRoles,omitempty"`
+
+	// SpaceRequestConfigName stores all the configuration related to the Space Request feature
+	// +optional
+	SpaceRequestConfigName *SpaceRequestConfig `json:"spaceRequestConfig,omitempty"`
+}
+
+// SpaceRequestConfig contains all the configuration related to the Space Request feature
+// +k8s:openapi-gen=true
+type SpaceRequestConfig struct {
+	// Provides the name of the Service Account whose token is to be copied
+	// +optional
+	SaTokenToCopy string `json:"saTokenToCopy,omitempty"`
```

Comment:
```suggestion
	ServiceAccountName string `json:"serviceAccountName,omitempty"`
```
`SaTokenToCopy` _may_ suggest that the field contains the actual token

Reply:
Yeah , updated